domains/integration-test/util.go

```diff
@@ -101,3 +159,33 @@ func newZetoDomain(t *testing.T, config *zetotypes.DomainFactoryConfig, factoryA
 	}
 	return waitForDomain, tbd
 }
+
+func findAvailableCoins[T any](t *testing.T, ctx context.Context, rpc rpcclient.Client, domainName, coinSchemaID, methodName string, address *tktypes.EthAddress, jq *query.QueryJSON, readiness ...func(coins []*T) bool) []*T {
```

Comment:
Rather than requiring an RPC method name to be passed in, should we consider one of these options:
- provide two helpers (one that queries states and another that queries nullifiers)
- use a boolean param to choose between states and nullifiers

Reply:
will make the update in my outstanding branch for adding locking support to the nullifier tokens